backend/main/workers/tracebacks.py

```diff
@@ -42,17 +42,27 @@ def didyoumean_suggestions(e) -> List[str]:
         log.exception("Failed to get didyoumean suggestions")
         return []
 
+def get_generic_explanation(e):
+    ft = FriendlyTraceback(type(e), e, e.__traceback__)
+    ft.compile_info()
```

Comment:
I need to look into it, but the name implies to me that this puts together lots of info, some of which I might not need. Does it? Either way, please refactor to only call this once.